### ISSUE TITLE
Only CIVET 1.1.10 for Ubuntu 12.04 works

### ISSUE THREAD

**Issue by pipitone** [TIGRLab/admin#144]
@awheel24 reports that at the moment CIVET/1.1.10 and CIVET/1.1.12 fail to submit jobs to the queue for various reasons.

**Comment by pipitone** [comment]
Recompiled CIVET/1.1.12 (fixed in tigrlab/quarantine@d4f3dd5) and have confirmed it works. Currently because of another issue with bash and environment-modules (will file it shortly, but until then see [here](https://rc.fas.harvard.edu/shellshock-update-issue-temporarily-affecting-slurm-jobs-software-modules/)) CIVET jobs don't run properly on the cluster. 

(Although, a very annoying workaround for the moment is to run `unset module` before starting the pipeline.)

**Comment by pipitone** [comment]
I can't seem to get CIVET/1.1.10 to compile again. Yay.

**Comment by josephdviviano** [comment]
Would it be safe to depreciate it?

**Comment by pipitone** [comment]
> Would it be safe to depreciate it?

We can just use the one that compiled on Ubuntu 12.04. :-)
